frontend/src/styles/element-dark.scss

```diff
@@ -460,4 +460,14 @@ html.dark {
     .el-checkbox__input.is-indeterminate .el-checkbox__inner::before {
         background-color: var(--panel-main-bg-color-10);
     }
+
+    .custom-input-textarea {
+        background-color: var(--panel-main-bg-color-10) !important;
+        color: var(--el-color-info) !important;
+    }
+    .custom-input-textarea:hover {
+        background-color: var(--panel-main-bg-color-9) !important;
+        color: var(--el-color-primary) !important;
+    }
+
 }
```

Comment:
There are no irregularities or immediate issues with the provided CSS changes. However, there are optimization suggestions that could be made to ensure better maintainability and performance:

1. **Namespace Classes**: Consider using a more namespace-like approach to separate different styles within one class instead of duplicating it with `.custom-input-textarea` and its hover state. This would make maintenance easier if additional styles are needed.

   ```css
   .custom-input {
       --background-color: var(--panel-main-bg-color-10);
       --color: var(--el-color-info);

       background-color: var(--background-color) !important;
       color: var(--color) !important;

       &.hover-state {
           --background-color: var(--panel-main-bg-color-9);
           --color: var(--el-color-primary);

           background-color: var(--background-color) !important;
           color: var(--color) !important;
       }
   }
   ```

2. **CSS Variables Recapitulation**: It might be beneficial to recapitalize the `--el-color-*` variables consistently throughout their uses in the dark theme rules for uniformity.

3. **Vendor Prefixing**: If these styles need to support older browsers specifically (not covered in your knowledge cutoff), consider adding vendor prefixes where necessary.

These suggestions aim to clean up the styling while maintaining clarity and functionality.